products/models.py

```diff
@@ -0,0 +1,124 @@
+from pydoc import describe
+from tkinter import CASCADE
+from django.db import models
+# from users import User
+
+class Base(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+class Main_Category(Base):
+    name = models.CharField(max_length=200)
+    
+    class Meta:
+        db_table = 'main_categories'
+
+class Sub_Category(Base):
+    name = models.CharField(max_length=200)
+    description = models.TextField()
+    main_category = models.ForeignKey(Main_Category, on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'sub_catogries'
+
+class Product(Base):
+    name = models.CharField(max_length=200)
+    description = models.TextField()
+    ingredients_etc = models.TextField()
+    sub_category = models.ForeignKey(Sub_Category)
+
+    class Meta:
+        db_table = 'products'
+
+class Product_Option(Base):
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
+    size = models.CharField(max_length=30)
+    price = models.PositiveIntegerField()
+
+    class Meta:
+        db_table = 'products_options'
+
+class Key_Ingredients(Base):
```

Comment:
class명은 단수, table명은 복수로 만들어주세요
```suggestion
class KeyIngredient(Base):
```